website/src/styles/header.css

```
@@ -71,8 +70,80 @@
+ * fits. The toggle is a checkbox so the menu works with no client script: the
+ * label is the button, and the checked state shows the links. The input is
+ * visually hidden rather than display:none so it keeps its place in the tab
+ * order, and the label shows its focus ring.
+ */
+.nav-menu-toggle {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)